background/app/commands-handling.js

```diff
@@ -21,7 +21,10 @@ define("background/app/commands-handling", function(require, exports, module) {
 		switch (command) {
 
 			case Commands.GOTO_LINE:
-				// Commands that should be passed to content script.
+				conn.notify(command);
+				break;
+
+			case Commands.GOTO_FILE : 
 				conn.notify(command);
 				break;
```

Comment:
My idea was that all cases when command should be passed to content script use the same conn.notify(...). Like this:
case Commands.GOTO_LINE:
case Commands.GOTO_FILE:
    conn.notify(command);
    break;
That way if something changes to sending command we can update on one place.